operator/src/main/java/org/bf2/operator/operands/KafkaInstanceConfiguration.java

```diff
@@ -388,6 +388,8 @@ public static class AccessControl {
         protected String finalVersion = "0.0.0"; // Default, disables `acl-legacy` configurations
         @JsonProperty("authorizer-class")
         protected String authorizerClass = null;
+        @JsonProperty("broker-plugins-config-prefix")
+        protected String brokerPluginsConfigPrefix = null;
         @JsonProperty("config-prefix")
         protected String configPrefix = null;
```

Comment:
Is this deprecated now?

Reply:
Ah yes, I guess so. I'll add the annotation to it now, thanks!